views/js/pciCreator/ims/mathEntryInteraction/creator/widget/states/Map.js

```diff
@@ -130,7 +130,7 @@ define([
             interaction = self.widget.element;
 
         interaction.onPci('responseChange', function (latex) {
-            if (self.inGapMode(self) === false && self.activeEditId !== null) {
+            if ((self.inGapMode(self) === false) && self.activeEditId !== null) {
```

Comment:
**chore:** The additional parenthesis are useless and could have been removed...